tortoise/algorithm.go

```diff
@@ -133,7 +133,8 @@ func (t *Tortoise) OnBeacon(eid types.EpochID, beacon types.Beacon) {
 		log.Uint32("evicted", evicted.Uint32()),
 		log.Stringer("beacon", beacon),
 	)
-	if eid <= evicted {
+	// when the node recover from checkpoint, the new effective genesis can be the same as the beacon genesis
```

Comment:
what is beacon genesis?

Reply:
sorry. i mean to say 
"the new effective genesis can be the target epoch for the beacon"